widgetastic_manageiq.py

```diff
@@ -1333,3 +1335,171 @@ def all_emails(self):
 
     def read(self):
         return list(self.all_emails)
+
+
+class TimelinesZoomSlider(View):
+    """This control represents Timeline's Zoom Slider
+
+    """
+    ROOT = ParametrizedLocator('{@locator}')
+    zoom_in_button = Text(locator='//button[@id="timeline-pf-zoom-in"]')  # button +
```

Comment:
**Optional**
Change the formatting a bit on the comment, the +/- aren't immediately clear. maybe something like `  # "+" button`